Renci.SshNet.Async/Renci.SshNet.Async.csproj

```diff
@@ -1,47 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;net40;uap10.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;</TargetFrameworks>
```

Comment:
Maybe some modern target framework could be also added? .NET 8 ?

Reply:
I second that.